### DISCUSSION TITLE
Welcome to Subsocial Web UI discussions!

### DISCUSSION THREAD

**Discussion by subsocialdev** [dappforce/subsocial-web-app#389] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arjdesign** [comment]
I was trying to install subsocial locally. I got stuck here:

Connect the app to Subsocial's server (Substrate node, IPFS):

Where is the `subsocial-v2.env` file?

**Reply by subsocialdev** [reply]
I just fixed the README file.
It should be `subsocial-betanet.env` instead.

**Reply by arjdesign** [reply]
Thank you!
